src/opera/api/controllers/outputs_controller.py

```diff
@@ -20,14 +22,7 @@ def get_outputs(session_token):
     """
 
     session_data = SQL_database.get_session_data(session_token)
```

Comment:
I think this is redundant since validation has been moved to security_controller.

Reply:
Agree